src/containers/tud/TimeUseDiaryReducer.js

```diff
@@ -71,32 +70,35 @@ export default function timeUseDiaryReducer(state :Map = INITIAL_STATE, action :
       });
     }
 
-    case downloadTudResponses.case(action.type): {
+    case downloadTudData.case(action.type): {
```

Comment:
I don't understand the difference between having 2 dedicated reducers and a single merged reducer with an if-check in every case ... just seems like you're mixing logic with no clear benefit.